nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestBridgeCommentManager.java

```diff
@@ -79,5 +87,22 @@ protected DocumentModel getCommentedDocument(){
         return BridgeCommentManager.class;
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * In the case of comment bridge we will ignore the check when we have an event of deletion comment, because the
+     * bridge checks the existence of the comment document and in this case we will end with an exception.
```

Comment:
Isn't a real production issue?

Reply:
you mean if the failure is a not really important ?

Reply:
I mean: could this happen in production?

Reply:
No i think it is just a test use case because i make this assertion `  assertEquals(commentManager.getCommentedDocumentRef(session, expectedDocModel), commentedDocModel.getRef());` and in the case of the bridge we get the document to be able on which implementation we root it. i will update the comment method